index.html

```diff
@@ -31,9 +31,16 @@
 					<img alt="White Flower Icon of Open Source Diversity" title="White Flower Icon" src="img/opensourcediversity-icon-white.svg" />
 					Open Source Diversity
 				</a>
-				<button class="navbar-toggler first-button" type="button" data-toggle="collapse" data-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation"></button>
-					<div class="animated-icon"><span></span></div>
-				
+				<button class="navbar-toggler first-button" type="button" data-toggle="collapse" data-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
+					<span class="screen-reader-text">Menu</span>
```

Comment:
is this a new thing? Seems like a good thing but I am not very familiar with this code. Not sure if this was related to a failing HTML linter or if seemed appropriate to add this in here?

Reply:
This is a re-implementation of hamburger button for menu in mobile width that got removed in #138 

This has to be handled to pass lint because the empty button and span tags trigger warnings from tidy.


I decided not to revert to old changes and implement it in a different way because this would be a more semantic HTML way to do this. 

Also, interestingly, this bug may have been introduced to fix lint errors (as described in #123). So, we're coming full circle.

Reply:
thanks for clarifying 🙏 I could certainly do with a refresh of the code base!